Dockerfile.tpl

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6-alpine
+MAINTAINER source{d}
+
+# XXX remove vim
```

Comment:
pip is needed to install the dependencies (six, pydetector) and the Python driver itself once copied. Removing bash and build-base.